mailpoet/tests/_support/AcceptanceTester.php

```diff
@@ -52,6 +52,13 @@ class AcceptanceTester extends \Codeception\Actor {
   const WOO_COMMERCE_MEMBERSHIPS_PLUGIN = 'woocommerce-memberships';
   const WOO_COMMERCE_SUBSCRIPTIONS_PLUGIN = 'woocommerce-subscriptions';
 
+  public function activateFeature(string $feature) {
+    $feature = strtolower($feature);
+    $this->importSql([
+      'insert into mp_mailpoet_feature_flags (name, value) values ("'. $feature . '",1)',
+    ]);
+  }
+
```

Comment:
We already have a data factory for that: https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/tests/DataFactories/Features.php

Reply:
Thank you for pointing that out. Funny, I didn't end up this method in the end. Was a residual and I forgot about it. I switched to the DataFactory in the `_before` method of the test itself. [b5b8999](https://github.com/mailpoet/mailpoet/pull/4434/commits/b5b8999931476b1fd98c53a0bac8f2535fa8c6ca)